Feature/Nicolas Parra/AudioManagerLevel1

Nik-Stevens opened the pull request:
Se creó un script que llama configura los audios, se agregaron 2 archivos de sonido para prueba
Se crearon dos GameObject, uno el AudioManager para llamar a las funciones de Audio y el segundo es El GameObject que contiene los audios

Nik-Stevens committed:
AudioManagerLevel1

Se incorporó el Audio Manager para manejar el sonido del primer nivel

Loko24 commented on Assets/Scripts/AudioManager.cs:
Fix else condition

Loko24 reviewed (changes requested):
Fix private variables with underscore before name it, fix else condition

Nik-Stevens committed:
Code Fixes

The underscore was added to comply with the nomenclature and  the method "ReproducirSonido" was changed to English "PlaySound"

Nik-Stevens commented:
I added the fixes 
The Else condition was eleminated and a Method was changed to English

Nik-Stevens committed:
[Uu]ser[Ss]ettings/

[Uu]ser[Ss]ettings/ Added to gitignore

Nik-Stevens committed:
AudioManagerLevel1

Conflicts should be fixed

Nik-Stevens committed:
Delete default-2022.dwlt

Nik-Stevens committed:
Merge branch 'dev' into Feature/NicolasParra/AudioManagerLevel1

JPulido01 reviewed (approved):
It's ready
